diff --git a/include/orm/tiny/tinytypes.hpp b/include/orm/tiny/tinytypes.hpp
index 8eb96878..e644f8e4 100644
--- a/include/orm/tiny/tinytypes.hpp
+++ b/include/orm/tiny/tinytypes.hpp
@@ -11,7 +11,7 @@ TINY_SYSTEM_HEADER
 
 #ifdef TINY_NO_INCOMPLETE_UNORDERED_MAP
 #  include <map>
- Leaving only for reference, the unordered_map is included in the modelscollection.hpp
+// Leaving only for reference, the unordered_map is included in the modelscollection.hpp
 #else
 #  include <unordered_map>
 #endif
