diff --git a/cmake/TinyOrmConfig.cmake.in b/cmake/TinyOrmConfig.cmake.in
index c7b0a61a..f7eee8f1 100644
--- a/cmake/TinyOrmConfig.cmake.in
+++ b/cmake/TinyOrmConfig.cmake.in
@@ -18,9 +18,9 @@ unset(tiny_build_type)
 message(CHECK_START "Configuring @PROJECT_NAME@ package")
 list(APPEND CMAKE_MESSAGE_INDENT "  ")
 
-set_and_check(@TinyOrm_ns@_BIN_DIR "@PACKAGE_BIN_INSTALL_DIR@")
+set(@TinyOrm_ns@_BIN_DIR "@PACKAGE_BIN_INSTALL_DIR@")
 set_and_check(@TinyOrm_ns@_CONFIG_DIR "@PACKAGE_CONFIG_INSTALL_DIR@")
-set_and_check(@TinyOrm_ns@_DOC_DIR "@PACKAGE_DOC_INSTALL_DIR@")
+set(@TinyOrm_ns@_DOC_DIR "@PACKAGE_DOC_INSTALL_DIR@")
 set_and_check(@TinyOrm_ns@_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
 set_and_check(@TinyOrm_ns@_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")
 
